lib/client.js

```diff
@@ -7,34 +7,38 @@ const co = require('co');
 
 const DEFAULT_EXCHANGE_TYPE = 'topic';
 const DEFAULT_HEARTBEAT = 10;
+const DEFAULT_RECONNECT_TIMEOUT = 1000;
 
 /**
  * Return a bus client with helper methods to communicate with an amqp server.
  *
  * @name  createClient.
  * @param {String} url : The url of your amqp server.
  * @param {Object} [options]
- * @param {Number} [options.heartbeat] : the heartbeat that you want to use.
+ * @param {Number} [options.heartbeat=10] : the heartbeat that you want to use.
+ * @param {Number} [options.reconnectTimeout=1000] : delay to reconnect broken connections
  */
 function* createClient(url, options) {
   options = options || {};
   options.heartbeat = options.heartbeat || DEFAULT_HEARTBEAT;
-  const connection = yield amqplib.connect(url, options);
-  const channel = yield connection.createChannel();
+  options.reconnectTimeout = options.reconnectTimeout || DEFAULT_RECONNECT_TIMEOUT;
```

Comment:
not such a big fan, I find this was simpler and it was already done like this before. Except if there is some benefit that I missed ?
If I didn't fear to break anything I would use ES6 destructuring with default rather than this